src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring.mdx

```diff
@@ -52,11 +52,9 @@ By default, our [Helm Chart](/docs/kubernetes-pixie/kubernetes-integration/insta
 
 #### `hostNetwork` and `privileged` [#hostnetwork-privileged]
 
-Most users and Kubernetes distributions configure the control plane metrics endpoints to listen only in the loopback interface, i.e. `localhost`. For this reason, the control plane component is deployed with `hostNetwork: true` by default when `privileged` is set to `true` (the default).
+In previous versions, when the integration is deployed using `privileged: false`, the `hostNetwork` setting for the control plane component will be also be set to `false`.
```

Comment:
I would try to avoid to specify new/latest versions since in the future it could be misleading.

Reply:
previous versions of v3 or you mean <v3?

Reply:
I agree with @paologallinaharbur. I think it'd be less ambiguous if we specified the versions where this happens.

Reply:
Should we unify this referrals ? And stop using relative terms previous/latest/news by more concrete terminology (v3, >v3 or <v3)

Reply:
Solved in the commit [#5272112](https://github.com/newrelic/docs-website/pull/7322/commits/52721120b118b37ff91856396a750b4964b520c4)